test/kubernetes/deploy/deploy.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if ! command -v kustomize &> /dev/null; then
```

Comment:
`&>` is Bash v4 only. Better to use `2>&1`. (See also [this SO answer](https://stackoverflow.com/a/876267/420061).)

Similar below.

Reply:
Aha, good catch. Fixed.